trezorlib/tests/device_tests/test_msg_cardano_sign_transaction.py

```diff
@@ -19,22 +19,35 @@
 import pytest
 
 from trezorlib import cardano, messages
+from trezorlib.exceptions import TrezorFailure
 
 from .conftest import setup_client
 
-VECTORS = [
+PROTOCOL_MAGICS = {
+    'mainnet': 764824073,
+    'testnet': 1097911063
+}
+
+EXAMPLE_INPUTS = [
+    {
+        "input": {
+            "path": "m/44'/1815'/0'/0/1",
+            "prev_hash": "1af8fa0b754ff99253d983894e63a2b09cbb56c833ba18c3384210163f63dcfc",
+            "prev_index": 0,
+            "type": 0,
+        },
+        "prev_tx": "839f8200d818582482582008abb575fac4c39d5bf80683f7f0c37e48f4e3d96e37d1f6611919a7241b456600ff9f8282d818582183581cda4da43db3fca93695e71dab839e72271204d28b9d964d306b8800a8a0001a7a6916a51a00305becffa0"
+    }
+]
+
+VALID_VECTORS = [
     # Mainnet transaction without change
     (
-        # protocol magic (mainnet)
-        764824073,
+        # protocol magic
+        PROTOCOL_MAGICS["mainnet"],
         # inputs
         [
-            {
-                "path": "m/44'/1815'/0'/0/1",
-                "prev_hash": "1af8fa0b754ff99253d983894e63a2b09cbb56c833ba18c3384210163f63dcfc",
-                "prev_index": 0,
-                "type": 0,
-            }
+            EXAMPLE_INPUTS[0]["input"]
```

Comment:
why not `[ i["input"] for i in EXAMPLE_INPUTS]` ? Similarly below. Though I wonder whether it wouldn't make sense to actually have two "example" values so that we don't need to extract these in each test...

Reply:
now we have a single example input, but maybe later we will have more of them and not all would be supposed to be used in a single test transaction